monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -313,6 +314,69 @@
                     "title": "Behaviour",
                     "type": "object",
                     "properties": {
+                        "custom_post_breach": {
+                            "title": "Custom post breach actions",
+                            "type": "object",
+                            "properties": {
+                                "linux": {
+                                    "title": "Linux command",
+                                    "type": "string",
+                                    "default": "",
+                                    "description": "Linux command to execute after breaching."
+                                },
+                                "linux_file": {
+                                    "title": "Linux file",
```

Comment:
linux_file and windows_file are used only to store the info that is displayed on front end.